Classes/Parser/ScssParser.php

```diff
@@ -83,9 +85,36 @@ protected function parseFile($file, $settings)
                 'sourceMapBasepath' => '<PATH DOES NOT EXIST BUT SUPRESSES WARNINGS>'
             ]);
         }
+        $absoluteFilename = GeneralUtility::getFileAbsFileName($file);
+        // Adds visual directory path of the initial file as import path
+        // This scenarios happens, when e.g. developing packages using the `path`
+        // repository feature of Composer - having one package in `public/typo3conf/ext/`
+        // and the other one symlinked in e.g. `packages/`.
+        // Since the PHP SCSS parser works on resolved real paths, the symlinked context is lost.
+        $visualImportPath = dirname($absoluteFilename);
+        $scss->addImportPath(function ($url) use ($visualImportPath) {
+            // Resolve potential back paths manually using PathUtility::getCanonicalPath,
+            // but make sure we do not break out of TYPO3 application path using GeneralUtility::getFileAbsFileName
+            // Also resolve EXT: paths if given
+            $isTypo3Absolute = !(strpos($url, '..') === 0);
```

Comment:
that is definitely a wrong check here now

Reply:
changed as discussed